python/matcher.go

```diff
@@ -39,16 +39,20 @@ func (*Matcher) Vulnerable(ctx context.Context, record *claircore.IndexRecord, v
 
 	v, err := pep440.Parse(record.Package.Version)
 	if err != nil {
+		zlog.Info(ctx).
```

Comment:
Thoughts on these being `warn`s?

Reply:
Yeah, okay. If these are this far into the system without anything noticing the versions are weird, that's more of a problem.